GAE/src/org/akvo/flow/rest/security/oidc/CallbackController.java

```diff
@@ -78,9 +78,15 @@ private void handle(HttpServletRequest req, HttpServletResponse res) throws IOEx
             SecurityContextHolder.getContext().setAuthentication(authentication);
             res.sendRedirect(redirectOnSuccess);
         } catch (AuthenticationException | IdentityVerificationException e) {
-            log.log(Level.SEVERE, e.getMessage());
-            SecurityContextHolder.clearContext();
-            res.sendRedirect(redirectOnFail);
+	    String errorDescriptionId = "error_description";
+	    String errorDescription= req.getParameter(errorDescriptionId);
+	    log.log(Level.SEVERE, errorDescriptionId+": "+errorDescription+" Error Message: "+e.getMessage());
```

Comment:
Do not log as SEVERE in the case of EMAIL_VERIFIED_ERROR. It is expected and it will cause a lot of noise.

Log as INFO for this particular case. Leave as SEVERE for other cases.

Reply:
Log as INFO for this particular case. Leave as SEVERE for other cases.